.prettierignore

```diff
@@ -9,4 +9,7 @@ dist/
 /packages/client/declaration
 /packages/client/scripts/default-index.js
 /reproductions
-.cache
\ No newline at end of file
+.cache
+node_modules/
```

Comment:
`node_modules/` is not really needed because it's already ignored by default by Prettier